.github/workflows/tests.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'stable/**'
```

Comment:
If this actually needs quoting (I don't think so), also quote the others for consistency.